Twig/Extension/CKEditorExtension.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Sonata\FormatterBundle\Twig\Extension;
+
+use Ivory\CKEditorBundle\Templating\CKEditorHelper;
+
+/**
+ * @author Marodon Jérémy <marodon.jeremy@gmail.com>
+ */
+class CKEditorExtension extends \Twig_Extension
+{
+    /**
+     * @var CKEditorHelper
+     */
+    private $helper;
+
+    /**
+     * Creates a CKEditor extension.
+     *
+     * @param CKEditorHelper $helper The CKEditor helper.
+     */
+    public function __construct(CKEditorHelper $helper)
+    {
+        $this->helper = $helper;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        $options = array('is_safe' => array('html'));
+
+        return array(
+            new \Twig_SimpleFunction('ckeditor_replace', array($this, 'renderReplace'), $options),
+        );
+    }
+
+    /**
+     * Renders the replace.
+     *
+     * @param string $id     The identifier.
+     * @param array  $config The config.
+     *
+     * @return string The rendered replace.
+     */
+    public function renderReplace($id, array $config)
+    {
+        @trigger_error('The ckeditor_replace twig tag is now deprecated and removed in 3.0, must be replace by ckeditor_widget.', E_USER_DEPRECATED);
```

Comment:
The 3.0 version is already released and the tag removed. If i understand the message is for 3.0 SonataFormatterBundle and not for IvoryCKEditorBundle bridge isn't it ? The meaning is will be removed in SonataFormatterBundle 3.0 ?